astropy/time/core.py

```diff
@@ -464,6 +499,35 @@ def __repr__(self):
     def __str__(self):
         return str(getattr(self, self.format))
 
+    def strftime(self, format_spec):
+        """
+        Convert Time to a string according to a format specification.
```

Comment:
This function converts to a string or numpy.array of strings.  Say so here and in the `Returns` section.